CoEpi/domain/model/KeyValueStore.swift

```diff
@@ -0,0 +1,26 @@
+import Foundation
+
+class KeyValueStore {
+    static let boolDefaults = ["showOnboarding": true]
+    
+    static func putBool(key: String, value: Bool) {
+        let defaultResult = boolDefaults[key]
+        if defaultResult == nil {
+            print("Error: Storing bool for invalid key: " + key)
+        } else {
+            UserDefaults.standard.set(value, forKey: key)
+        }
+    }
+    
+    static func getBool(key: String) -> Bool {
```

Comment:
I forgot to comment, don't use `static` here. The idea is to allow this component to be mocked. We can't do this with `static`.

Reply:
No longer using static methods.